### PR TITLE
[minor] remove usage of Kernel::$rootDir from docs

### PR DESCRIPTION
**Pull request by jzawadzki**
As per https://github.com/symfony/symfony/pull/28810 $rootDir has been removed in 5.0 - use getProjectDir in docs instead - small change to be consistent

### CODE REVIEW

File: reference/configuration/kernel.rst [hunk 1]
```diff
@@ -87,7 +87,7 @@ method to return the right project directory::
 Cache Directory
 ~~~~~~~~~~~~~~~
 
-**type**: ``string`` **default**: ``$this->rootDir/cache/$this->environment``
+**type**: ``string`` **default**: ``$this->getProjectDir()/var/cache/'$this->environment``
```
Comment (javiereguiluz) [inline reference/configuration/kernel.rst]: We should remove the extra ``'`` before `$this->environment`

**Review by javiereguiluz — APPROVED** [review]
Nice catch! Thanks.

**Review by wouterj — APPROVED** [review]
Great catch!

As this method is already available in 4.x, let's merge this into 4.3. You don't have to do anything for this (our merge tool can take care of it).

**Comment by javiereguiluz** [comment]
Thanks! (merged in 4.3)

**Pull request merged**